settings.gradle.kts

```diff
@@ -33,6 +33,7 @@ include(":aws-xray")
 include(":aws-xray-propagator")
 include(":baggage-processor")
 include(":compressors:compressor-zstd")
+include("cloudfoundry-resource")
```

Comment:
@trask: Do you want the entire project name changed to `cloudfoundry-resources`?